03_launch_mgmt_cluster.sh

```diff
@@ -252,7 +252,7 @@ function make_bm_hosts() {
 function apply_bm_hosts() {
   pushd "${BMOPATH}"
   list_nodes | make_bm_hosts > "${WORKING_DIR}/bmhosts_crs.yaml"
-  if [ "${EPHEMERAL_CLUSTER}" != "tilt" ]; then
```

Comment:
I've removed this check because `apply_bm_hosts` is only called from a place where `"${EPHEMERAL_CLUSTER}" != "tilt"` is already true.